### ISSUE TITLE
Need null safety support

### ISSUE THREAD

**Issue by Basher7** [fluttercommunity/flutter_google_places#154]
**from flutter v2.0.2, flutter is now null-safety mechanism. Most of the packages already upgrade to null safety and more are upgrading.**

Thanks for this great package!

_Do you plan to make a null safe version of the package?_

**Comment by hoc081098** [comment]
# Just published as new name: https://pub.dev/packages/flutter_google_places_hoc081098. Only replace the importing
```diff
- import 'package:flutter_google_places/flutter_google_places.dart';
+ import 'package:flutter_google_places_hoc081098/flutter_google_places_hoc081098.dart';
```

**Comment by KevMorelli** [comment]
Implemented on v0.3.0
